pkg/server/services/cluster.go

```
@@ -0,0 +1,116 @@
+			return err
+		}
+
+		_, err := c.clusterClient.ClusterV1().ManagedClusters().Update(ctx, cluster, metav1.UpdateOptions{})
+		return err
+	default:
+		return fmt.Errorf("unsupported cluster event action %s", eventType.Action)
+	}
+}
+
+func (c *ClusterService) RegisterHandler(handler server.EventHandler) {
+	c.clusterInformer.Informer().AddEventHandler(&cache.ResourceEventHandlerFuncs{
```

golangci-lint (1.64.8)

Error return value of `(k8s.io/client-go/tools/cache.SharedInformer).AddEventHandler` is not checked

(errcheck)